### ISSUE TITLE
Failed to generate thumbnails

### ISSUE THREAD

**Issue by ranjanprj** [mirumee/saleor#3640]
### What I'm trying to achieve
…
I'm running Saleor 2.1.0 in docker componse environment
### Steps to reproduce the problem
1. Run the Saleor 2.1.0 in Win10 - Docker Compose with Hyper-V env.
2. The application works fine but keeps giving following errors

```
celery_1  | INFO saleor.core.utils Creating thumbnails for  9 [PID:10:MainThread]
celery_1  | [2019-01-23 04:19:06,655: INFO/ForkPoolWorker-1] Creating thumbnails for  9
celery_1  | [2019-01-23 04:19:06,657: INFO/ForkPoolWorker-2] Task saleor.product.thumbnails.create_category_background_image_thumbnails[f9cc8406-75f4-44e1-82da-4b5a6f822b53] succeeded in 0.037414300000023104s: None
celery_1  |
celery_1  | [2019-01-23 04:19:06,670: ERROR/ForkPoolWorker-1] Thumbnail generation failed
celery_1  | Traceback (most recent call last):
celery_1  |   File "/usr/local/lib/python3.6/site-packages/versatileimagefield/image_warmer.py", line 117, in _prewarm_versatileimagefield
celery_1  |     url = get_url_from_image_key(versatileimagefieldfile, size_key)
celery_1  |   File "/usr/local/lib/python3.6/site-packages/versatileimagefield/utils.py", line 216, in get_url_from_image_key
celery_1  |     img_url = img_url[size_key].url
celery_1  |   File "/usr/local/lib/python3.6/site-packages/versatileimagefield/datastructures/sizedimage.py", line 149, in __getitem__
celery_1  |     height=height
celery_1  |   File "/usr/local/lib/python3.6/site-packages/versatileimagefield/datastructures/sizedimage.py", line 201, in create_resized_image
celery_1  |     path_to_image
celery_1  |   File "/usr/local/lib/python3.6/site-packages/versatileimagefield/datastructures/base.py", line 140, in retrieve_image
celery_1  |     image = self.storage.open(path_to_image, 'rb')
celery_1  |   File "/usr/local/lib/python3.6/site-packages/django/core/files/storage.py", line 33, in open
celery_1  |     return self._open(name, mode)
celery_1  |   File "/usr/local/lib/python3.6/site-packages/django/core/files/storage.py", line 218, in _open
celery_1  |     return File(open(self.path(name), mode))
celery_1  | FileNotFoundError: [Errno 2] No such file or directory: '/app/media/category-backgrounds/cos_iyNE7C2.jpg'
celery_1  | ERROR saleor.core.utils Failed to generate thumbnails [PID:10:MainThread]
celery_1  | [2019-01-23 04:19:06,670: ERROR/ForkPoolWorker-1] Failed to generate thumbnails

### What I expected to happen
```
…

### Screenshots

![image](https://user-images.githubusercontent.com/774370/51599950-dab4a180-1f26-11e9-9bf8-3d234a584ef9.png)

**System information**
Operating system: Windows 10 
Browser: Brave Browser

**Comment by michaljelonek** [comment]
Same as https://github.com/mirumee/saleor/issues/3548. We are currently working on a fix for this. The issue is that `populatedb` is run on `web` container, which doesn't mount the app, generating all the images, while celery can't find them (as they're living on the web container).

**Comment by maarcingebala** [comment]
Should be resolved by #3657
